godaddy.com.waf.json

```diff
@@ -0,0 +1,27 @@
+{
+  "providerId": "godaddy.com",
+  "providerName": "GoDaddy",
+  "serviceId": "waf",
+  "serviceName": "Web Application Firewall",
+  "version": 2,
+  "logoUrl": "https://www.domainconnect.org/wp-content/uploads/2020/02/godaddy_logo_new.png",
+  "description": "Enables web application firewall for GoDaddy protection product.",
+  "variableDescription": "",
```

Comment:
Please add variableDescription.  See following for example:  https://github.com/Domain-Connect/Templates/blob/master/godaddy.com.plesk.json

Reply:
Thanks @wdbaker-godaddy!